### PR TITLE
Map tasks2.0

### PR DESCRIPTION
**Pull request by Arens404**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        HashMap<String, String> telephoneNumbers = new HashMap<String, String>();
+        telephoneNumbers.put("William A. Lathan", "405-709-1865");
+        telephoneNumbers.put("John K. Miller", "402-247-8568");
+        telephoneNumbers.put("Hortensia E. Foster", "606-481-6467");
+        telephoneNumbers.put("Amanda D. Newland", "319-243-5613");
+        telephoneNumbers.put("Brooke P. Askew", "307-687-2982");
+        System.out.print("Select key:");
+        String keySelect = scanner.nextLine();
```
Comment (Chezo) [inline Main.java]: use meaningful name which is not a verb

**Review by Chezo — CHANGES_REQUESTED** [review]

File: Main.java (hunk 1 above)
Comment (Arens404) [reply Main.java]: 4388e443

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        Scanner scanner = new Scanner(System.in);
+        HashMap<String, String> telephoneNumbers = new HashMap<String, String>();
+        telephoneNumbers.put("William A. Lathan", "405-709-1865");
+        telephoneNumbers.put("John K. Miller", "402-247-8568");
+        telephoneNumbers.put("Hortensia E. Foster", "606-481-6467");
+        telephoneNumbers.put("Amanda D. Newland", "319-243-5613");
+        telephoneNumbers.put("Brooke P. Askew", "307-687-2982");
+        System.out.print("Select key:");
+        String key = scanner.nextLine();
+        System.out.println("(Press P/N)");
```
Comment (Chezo) [inline Main.java]: for what? xD

**Review by Chezo — CHANGES_REQUESTED** [review]

File: Main.java (hunk 2 above)
Comment (Arens404) [reply Main.java]: f8afd403

Comment (Chezo) [reply Main.java]: still "     System.out.println("(Press P/N)");" FOR WHAT?

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        HashMap<String, String> telephoneNumbers = new HashMap<String, String>();
+        telephoneNumbers.put("William A. Lathan", "405-709-1865");
+        telephoneNumbers.put("John K. Miller", "402-247-8568");
+        telephoneNumbers.put("Hortensia E. Foster", "606-481-6467");
+        telephoneNumbers.put("Amanda D. Newland", "319-243-5613");
+        telephoneNumbers.put("Brooke P. Askew", "307-687-2982");
+        System.out.print("Select key:");
+        String keyForGetValue = scanner.nextLine();
```
Comment (Chezo) [inline Main.java]: too log name

**Review by Chezo — CHANGES_REQUESTED** [review]

File: Main.java (hunk 2 above)
Comment (Arens404) [reply Main.java]: 5b28fa1b

File: Main.java (hunk 3 above)
Comment (Arens404) [reply Main.java]: fac30b60

**Review by Chezo — APPROVED** [review]

**Pull request merged**